foundations/installations/installations.md

```diff
@@ -261,14 +256,106 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
+- Enter the following command
+
+~~~powershell
+wsl --install
+~~~
+
+- After a few minutes you'll be prompted to enter a new username. This username should be lowercase, but can otherwise be whatever suits you. You'll also need to enter a new password.
+
+#### Step 1.3.1: Install Windows Terminal (Windows 10 only)
+
+Windows Terminal is an terminal application that lets you more easily customize and run terminals, as well as supporting multiple tabs which can each run their own different terminals.
+
+- Follow [Microsoft's guide](https://learn.microsoft.com/en-us/windows/terminal/install) to install Windows Terminal.
+
+#### Step 1.3.2: Setting WSL as default (Optional)
+
+Unless you are regularly using other terminals on your computer we recommend that you set WSL as the default terminal program when you open Windows Terminal. You can follow the [section on setting a default profile](https://learn.microsoft.com/en-us/windows/terminal/install#set-your-default-terminal-application) in the guide above to set it to Ubuntu.
+
+### Step 2 Opening WSL
+
+On Windows there are two primary ways to open WSL: either via the terminal or via the app icon.
+
+- If you set Windows Terminal to open a Ubuntu terminal by default you can start a new WSL session by simply opening the terminal app.
+- If you search for Ubuntu in the application search bar you should see a application titled Ubuntu; simply open it to start a new terminal session.
+
+<div class="lesson-note lesson-note--warning">
+When opening WSL ensure that you do not see /mnt/c at the start of the line. /mnt/c is where your Windows installation lives when working within WSL, and messing around there can have unintended consequences
+</div>
+
+### Step 3: WSL and systemd
+
+`systemd` is a system and service manager for Linux that you'll use much later in the curriculum. While it doesn't work on WSL by default, you can enable it.
+
+- Open a new WSL terminal window.
+- Enter the following command:
+
+~~~bash
+sudo nano /etc/wsl.conf
+~~~
+
+- When prompted for a password enter the password you set earlier. You may notice that while entering your password you don't see any visual feedback. This is an intentional security measure, and nothing to worry about.
+- You'll be shown `nano`, a text editor for Linux. Enter the following two lines
+
+~~~
+[boot]
+systemd=true
+~~~
+
+- It should now look like this:
+
+![Nano window](installations/imgs/21_nano_systemd.png)
+
+- Press <kbd>Ctrl</kbd> + <kbd>O</kbd>, then when prompted with which file name to write to press <kbd>Enter</kbd>, and then press <kbd>Ctrl</kbd> + <kbd>X</kbd> to exit `nano`
+- Close all open WSL terminal windows
+- Run the following command in Powershell (**not** WSL) to restart WSL
+
+~~~powershell
+wsl.exe --shutdown
+~~~
+
+- `systemd` should now be working. You can test this by running `systemctl list-unit-files --type=service`, which should return a long list of services.
```

Comment:
1. We are still inside the `PowerShell`, should be noted to run this command in a new WSL/Ubuntu terminal
2. Add a note on how to close the output with <kbd>Ctrl</kbd> + <kbd>z</kbd>

Reply:
can't find this section anymore

Reply:
See above